docs/swarm/swarm-tutorial/index.md

```diff
@@ -50,16 +50,26 @@ provider. This tutorial uses the following machine names:
 
 ###  Docker Engine 1.12 or later
 
-To use swarm mode, you must [install Docker Engine](../../installation/index.md)
-on each one of the host machines. Alternatively, install the latest Docker for
-Mac or Docker for Windows.
+This tutorial requires that you are running Docker Engine 1.12 or later on each of the host machines.
 
->**Note**: Docker for Mac and Docker for Windows let you use single-node
-features of swarm mode, like creating a swarm and creating a service. Multi-node
-features like joining additional nodes and scaling a service are not available.
+You can get Docker Engine v1.12 by installing it directory on Linux based systems, or with Docker for Mac or Docker for Windows.
```

Comment:
I'm unsure whether it makes sense to mention Docker for Mac and Windows here, since you can't really get the docker binary that way.

I'd recommend:

 1. If you just want to test on one node, know that you can do that directly with Docker for Mac and Win. `docker swarm init` and then skip to step X
 2. Or you can set up 3 hosts. The easiest is to use `docker-machine`, that way Docker is also installed for you (at this point perhaps also link to the labs scripts if people want easier option: https://github.com/docker/labs/tree/master/swarm-mode)
 3. Or you can do everything yourself